src/internal/kopia/upload.go

```diff
@@ -189,7 +189,7 @@ func (cp *corsoProgress) FinishedFile(relativePath string, err error) {
 
 	var locationFolders string
 	if d.locationPath != nil {
-		locationFolders = d.locationPath.Folder()
+		locationFolders = d.locationPath.Folder(false)
```

Comment:
this should be true 😝 
```suggestion
		locationFolders = d.locationPath.Folder(true)
```

Reply:
false